File: cpp/32e_1850A_ToMyCritics.cpp
#include<bits/stdc++.h>

int main(){
    int t, a, b, c, sum;
    std::vector<std::string> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>a>>b>>c;
        sum = 0;
        sum = a + b + c;
        if(sum - a >= 10){
            ans.push_back("YES");
        }
        else if(sum - b >= 10){
            ans.push_back("YES");
        }
        else if (sum - c >= 10){
            ans.push_back("YES");
        }
        else{
            ans.push_back("NO");
        }
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/32e_1850A_ToMyCritics.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        a, b, c = input().split()
        a, b, c = int(a), int(b), int(c)
        Sum = 0
        Sum = a + b + c
        if Sum - a >= 10:
            ans.append("YES")
        elif Sum - b >= 10:
            ans.append("YES")
        elif Sum - c >= 10:
            ans.append("YES")
        else:
            ans.append("NO")
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()